src/pages/_LoadingOverlay.jsx

```diff
@@ -15,7 +15,11 @@ const LoadingPage = () => {
     return () => { isActive = false; };
   }, []);
 
-  return <ApplicationLoadingOverlay isOpen={state.isOpen} />;
+  return (
+    <div data-terra-dev-site-loading data-terra-test-loading>
```

Comment:
Can these data attributes be applied on the overlay directly?

Reply:
Maybe? I feel more comfortable adding them here like this though. It's a temporary div shown during loading.